packages/engine-server/src/drivers/file/storev2.ts

```diff
@@ -199,6 +200,13 @@ export class FileStorage implements DStore {
     }
   }
 
+  /**
+   * See {@link DStore.getAllNotes}
+   */
+  async getAllNotes(): Promise<NotePropsByIdDict> {
```

Comment:
this seems much more expensive then return `this.notes`. 
since this is used a lot - can we get some benchmarks of how this performs, latency wise, in org-workspace?

Reply:
I added this to unblock future clients that require the entire engine notes and to avoid usages of `engine.notes`. Most should be using `getNote`.

I'll add some benchmarks for this to see its impact

Reply:
For 24695 notes in org-workspace, `getAllNotes` takes ~634ms

Reply:
634 ms may be non-trivial, depending on the type of operation.  Do we have any active references to this new `getAllNotes` method yet?

Reply:
None now, but after talking to @kevinslin offline, I'm going to remove this for now. Ideally most clients shouldn't need to get all notes so I think we can do a sweep first and provide methods on the engine/store side.

Looking at some current usages, its cases of getting all notes and doing own custom filtering so those can be refactored first